Apps/Sandcastle/gallery/3D Tiles 1.1 Photogrammetry.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
+    <!-- Use Chrome Frame in IE -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Comparison of tilesets from Cesium ion's current and next-gen 3D model tiling pipelines to showcase GPU memory improvements."
```

Comment:
Should the description be more oriented towards 3D Tiles 1.1 rather than the tiling pipeline?

Reply:
oops yes!